client/client.go

```diff
@@ -15,14 +15,14 @@ import (
 
 const (
 	apiKeyHeader = "X-API-Key"
-	apiKeyValue  = "l7xx8d8bfce4ee874269bedc02832674129b"
+	apiKeyValue  = "l7xx12ebcbc825eb480faa276e7f192d98d1"
 
 	userAgentHeader = "User-Agent"
 	userAgentValue  = "Southwest/3.0.26 (iPhone; iOS 9.1; Scale/2.00)"
 )
 
 var (
-	apiExtensionsBaseUrl = parseUrlOrPanic("https://api-extensions.southwest.com")
+	apiExtensionsBaseUrl = parseUrlOrPanic("https://mobile.southwest.com")
```

Comment:
It seems like this will work if you also add the following header to the request (around line 77):

```
httpReq.Header.Add("host", "mobile.southwest.com")
```

Reply:
Hmm I just tried it but I run into the same problem.  It works the first time that I run it, but if I want to run it again, I have to reinstall/recompile it.

Reply:
I think the error is from the API being called, not the code. I occasionally tried the same request using curl at various points, and most of the time it succeeded but occasionally got 400s back. I suspect either the API is attempting to subtly rate limit, or it's flaky.